packages/react-i18n/src/i18n.ts

```diff
@@ -65,17 +78,14 @@ export default class I18n {
     return this.languageDirection === LanguageDirection.Ltr;
   }
 
-  get countryCode(): string | undefined {
-    return this.locale.split('-')[1];
-  }
-
   constructor(
     public translations: TranslationDictionary[],
-    {locale, currency, timezone}: I18nDetails,
+    {locale, currency, timezone, pseudolocalize = false}: I18nDetails,
   ) {
-    this.locale = locale.toLowerCase();
```

Comment:
we don't need to lowercase it anymore?

Reply:
No, this was basically a bug/ incorrect choice in the previous version, an appropriate locale representation has lowercase language and uppercase region